src/main/java/net/dv8tion/jda/api/events/guild/update/GuildUpdateBannerEvent.java

```diff
@@ -81,4 +96,20 @@ public String getOldBannerUrl()
     {
         return previous == null ? null : String.format(Guild.BANNER_URL, guild.getId(), previous, previous.startsWith("a_") ? "gif" : "png");
     }
+
+    /**
+     * Returns an {@link ImageProxy} for this guild's old banner
+     * <p>
+     * <b>Note:</b> the old banner may not always be downloadable as it might have been removed from Discord
```

Comment:
```suggestion
     * Returns an {@link ImageProxy} for this guild's old banner.
     * <p>
     * <b>Note:</b> the old banner may not always be downloadable as it might have been removed from Discord.
```